README.md

```diff
@@ -1,2 +1,4 @@
 # utils
 A place for all the utility functions and helper functions. Most of this are pretty broad in the scope that they can be used in. Import these into your projects to have standardized utility functions.
+
+updated the README!!
```

Comment:
Sentences should start with uppercase letters.

Reply:
completed, please check again